<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CPX-85J1I2YMCLF" ignored="0" name="Login test with invalid credentials." tests="1" failures="1" timestamp="14 marraskuuta 2017 13:18:46 GMT" time="18.611" errors="0">
  <testcase name="Login_test_with_invalid_credentials" time="11.05" classname="tests.Login_test_invalid_credentials">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at pages.Login_Page.Check_welcome_text(Login_Page.java:58)
	at tests.Login_test_invalid_credentials.Login_test_with_invalid_credentials(Login_test_invalid_credentials.java:52)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Login_test_with_invalid_credentials -->
</testsuite> <!-- Login test with invalid credentials. -->
